ts/widget.ts

```diff
@@ -90,4 +86,44 @@ async function render({ model, el }: RenderProps<WasmWidgetModel>) {
   });
 }
 
-export default { render };
+export default async () => {
+  let wasmBinary: ArrayBuffer;
+  return {
+    async initialize({ model }: InitializeProps<WasmWidgetModel>) {
+      wasmBinary = await loadWasm(model);
```

Comment:
Render is called any time the widget instance is displayed. Since this is an asset, we and load it once. It might even make sense to have a global.